iohk/iohk-ops.hs

```diff
@@ -165,8 +164,8 @@ centralCommandParser =
      --                           (Nixops
      --                            <$> (NixopsCmd <$> argText "CMD" "Nixops command to invoke")
      --                            <*> ???)) -- should we switch to optparse-applicative?
-     ("create",                 "Create the whole cluster",                                         pure Create)
-   , ("modify",                 "Update cluster state with the nix expression changes",             pure Modify)
+     ("modify",                 "Update cluster state with the nix expression changes",             pure Modify)
+   , ("create",                 "Same as modify",                                                   pure Modify)
```

Comment:
Should we just remove `create`? I see the internal change happened at 605a4e4.

Reply:
Right, missed that..

Reply:
Ah, I didn't miss that, actually, it's an alias to modify, precisely for backward compat -- already forgot this..